k8s/testground-daemon/deployment.yml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testground-daemon
+  labels:
+    app: testground-daemon
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testground-daemon
+  template:
+    metadata:
+      labels:
+        app: testground-daemon
+    spec:
+      serviceAccountName: testground-daemon
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: goproxy
```

Comment:
@coryschwartz here we create a goproxy container, and both containers in this deployment connect to the host network and can communicate over localhost.

If goproxy had a service on Kubernetes, the `build container` (which is started from the `testground daemon`) would not be able to access it.